OpenSim/Simulation/Model/Frame.h

```diff
@@ -126,15 +134,25 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     const SimTK::Transform&
         getTransformInGround(const SimTK::State& state) const;
 
-    /** The spatial velocity V_GF {omega; v} for this Frame in ground.
+    /** The spatial velocity V_GF {omega; v} of this Frame in ground.
         It can be used to compute the velocity of any stationary point on F,
         located at r_F (Vec3), in ground, G, as:
             v_G = V_GF[1] + SimTK::cross(V_GF[0], r_F);
         Is only valid at Stage::Velocity or higher. */
     const SimTK::SpatialVec&
         getVelocityInGround(const SimTK::State& state) const;
 
-    /** The spatial acceleration A_GF {alpha; a} for this Frame in ground.
+    /** The angular velocity of this Frame in ground.
+        @see getVelocityInGround() */
+    const SimTK::Vec3&
+        getAngularVelocityInGround(const SimTK::State& state) const;
+
+    /** The linear velocity of this Frame in ground.
```

Comment:
You want me to say explicitly that this method returns the second chunk of the `SimTK::SpatialVec` returned by `getVelocityInGround()`?